src/commands/core/asset/create.ts

```
@@ -22,206 +22,341 @@
+    })
+    jsonSpinner.succeed(`Metadata uploaded to ${jsonUri}`)
 
-      // adjust json and upload
+    return jsonUri
+  }
 
-      const jsonFile = JSON.parse(fs.readFileSync(flags.json, 'utf-8'))
+  private formatAssetResult(result: any, explorer: ExplorerType): string {
+    return `--------------------------------
+  Asset: ${result.asset}
+  Signature: ${base58.deserialize(result.signature! as Uint8Array)[0]}
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)